pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_publishing-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,35 @@
+include::../attributes.adoc[]
+[id="publishing-a-module-in-pantheon_{context}"]
+= Publishing a module in {ProductShortName}
+
+Publish a module in {ProductShortName} to the Customer Portal.
+
+.Prerequisites
+
+* You are assigned the publisher role in {ProductShortName}.
+* You are logged in.
+* The module to be published has been uploaded to {ProductShortName}.
+
+.Procedure
+
+. Navigate to link:{LinkToSearchPage}[Search page].
+
+. Click the name of the module you want to publish.
++
+If you don't see the module listed, search for the modules using the search box.
+
+. In the list of module versions, click  *Publish* next to the version of the module you want to publish.
++
+Once published, the module can be accessed at the URL of the product it is published for, followed by its vanity URL fragment.
```

Comment:
Passive voice, and can be more concise. Change to something like "the module's URL is `https://__PRODUCT_URL__/__VANITY_URL_FRAGMENT__

(Using all caps for variables is based on current style council direction--not yet ratified by department.)

Reply:
Yes, all caps makes sense. Also, I'm not certain if this is true CCS-wide but in EAP docs we use italics for replaceable words. Let me know if the change makes sense.

Reply:
Yes, italics is the current style. The style council proposal is italics and all caps.

Reply:
From commit:

Once published, the module can be accessed at the following URL: =>
After you publish the module, you can access it at the following URL

Reply:
I do see the change in the variable formatting.

Reply:
Once published, the module can be accessed at the following URL: =>
After you publish the module, you can access it at the following URL

Done

Reply:
LGTM